earth_enterprise/docker/bin/build-docker-image.sh

```diff
@@ -1,33 +1,46 @@
 #! /bin/bash
 
-# Set these variables before running this script to override the
-# default values below:
-: ${STAGE_1_IMAGE_NAME:="opengee-experimental-stage-1"}
-: ${STAGE_2_IMAGE_NAME:="opengee-experimental-stage-2"}
-: ${OUTPUT_IMAGE_NAME:="opengee-experimental"}
+# This script builds a Docker image containing Open GEE Server and Fusion.
+
+source "$(dirname "$0")/lib/input-variables.sh"
```

Comment:
Doesn't this double quoting, in fact, leave $0 unquoted?

Reply:
No, it doesn't.  You could write a short script to test it for yourself, or dig through documentation on command substitution.

This specific expression would fail to produce the correct output, if the current directory name ends in one or more newline characters.  I've considered that an acceptable limitation.